.kokoro/release.sh

```diff
@@ -1,6 +1,60 @@
 #!/bin/bash
 
-# Make sure `JAVA_HOME` is set.
-echo "JAVA_HOME=$JAVA_HOME"
+# Get secrets from keystore and set and environment variables.
+setup_environment_secrets() {
+  export SONATYPE_USERNAME=functions-framework-release-bot
+  export SONATYPE_PASSWORD=$(cat ${KOKORO_KEYSTORE_DIR}/75669_functions-framework-java-release-bot-sonatype-password)
+  export GPG_PASSPHRASE=$(cat ${KOKORO_KEYSTORE_DIR}/75669_functions-framework-java-release-bot-gpg-passphrase)
+
+  # Add the keybox file to $GNUPGHOME to verify the GPG credentials.
+  export GNUPGHOME=/tmp/gpg
+  mkdir $GNUPGHOME
```

Comment:
Should we use this https://github.com/GoogleCloudPlatform/appengine-java-standard/blob/main/kokoro/gcp_ubuntu/release.sh#:~:text=%2DDgpg.homedir%3D%24%7BGNUPGHOME%7D

Reply:
Updated. My local test worked without the flags, but it might be needed to twist Kokoro mvn configs